### PR TITLE
Child 1

### PR DESCRIPTION
**Pull request by jaxinye**
pull request by owner, edit1

### CODE REVIEW

**Comment by jaxinye** [comment]
comment1

**Review by jaxinye — COMMENTED** [review]
review the request and make comment, 123

**Review by ghost — APPROVED** [review]
approve changes

**Review by ghost — CHANGES_REQUESTED** [review]
require a change

**Review by ghost — APPROVED** [review]

**Review by ghost — DISMISSED** [review]
1111

File: .gitmodules [hunk 1]
```diff
@@ -0,0 +1,3 @@
+[submodule "Python-100-Days"]
+	path = Python-100-Days
+	url = https://github.com/jaxinye/Python-100-Days.git
```
Comment (jaxinye) [inline .gitmodules:3]: comment on unified diff

File: child1.py [hunk 2]
```diff
@@ -0,0 +1,80 @@
+#imports
+import re
```
Comment (jaxinye) [inline child1.py:2]: start a review on unified diff

File: .gitmodules (hunk 1 above)
Comment (gitprojecttest1) [reply .gitmodules:3]: 333

File: child1.py [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+from urllib.request import urlopen #py 3
+#import urllib
+
+from bs4 import BeautifulSoup
```
Comment (jaxinye) [inline child1.py:6]: 44444

File: child1.py (hunk 2 above)
Comment (jaxinye) [inline child1.py:2]: 555555

**Review by jaxinye — COMMENTED** [review]
6666666

File: .gitmodules [hunk 4]
```diff
@@ -0,0 +1,3 @@
+[submodule "Python-100-Days"]
+	path = Python-100-Days
```
Comment (gitprojecttest1) [inline .gitmodules:2]: change requested on the line

**Review by ghost — DISMISSED** [review]
change

**Pull request merged**